src/collections/programs/gsoc-2019/index.mdx

```diff
@@ -0,0 +1,67 @@
+---
+
+import { ProgramsWrapper } from "../Programs.style.js";
+import Button from "../../../reusecore/Button";
+import gsoc_logo from "./gsoc.svg";
+
+<ProgramsWrapper>
+<h2>Google Summer of Code 2019 and Layer5 Community</h2>
+
+<div>
+
+<div className="flex-div">&nbsp;</div>
+    <div id="layer5-intro" className="iframe-container">
+        <iframe
+            width="392"
+            height="220.5"
+            src="https://www.youtube.com/embed/0yN5T5LB9ps"
+            frameBorder="0"
+            allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+            allowFullScreen />
+    </div>
+    <div>
+    <ul>
+        <b>What is Layer5 Community?</b>
+        <p>While small, the Layer5 community represents the largest collection of service mesh projects and their maintainers in the world. We build projects to provide learning environments, deployment and operational best practices, performance benchmarks, create documentation, share networking opportunities, and more. 
+        Our shared commitment to the open source spirit pushes Layer5 projects forward. New members are always welcome. </p>
+        <b>Is it Open Source?</b>
+        <p>Layer5 projects are open source software. Anyone can download, use, work on, and share it with others. 
+        It's built on principles like collaboration, globalism, and innovation. Layer5 projects are distributed under the terms of Apache v2. </p>
+        <b>Google Summer of Code Participation?</b>
+        <p>The key component of these projects is our Community. This community, which you will join as an participant in Google Summer of Code, is improving the world of diverse cloud native systems. 
+        Your contributions will affect people you've never met. The Layer5 community includes software engineers, researchers, students, artists, system administrators, operators and web designers -- all of whom will be happy to help you get started. </p>
+        <p>We believe that all contributors should expect and be part of a safe and friendly environment for constructive contribution. We can more effectively and successfully compare and challenge different ideas to find the best solutions for advancement, while building the size, diversity, and strength of our community. </p>
+    </ul>
+    </div>
+    <div>
+    <h3>Project Ideas</h3>
+        <ul>
+            <b>Meshery</b>
+            <p>Meshery is the service mesh management plane for lifecycle, configuration and performance management of many service meshes. See <a href="https://layer5.io/meshery">Meshery</a> for more information.</p>
+            <b>Linkerd and Envoy</b>
+            <p>Linkerd is an ultralight service mesh for Kubernetes and beyond: <a href="https://linkerd.io">https://linkerd.io</a> . Envoy is an open source edge and service proxy, designed for cloud-native applications: <a href="https://www.envoyproxy.io">https://www.envoyproxy.io</a></p>.
+            <b>Benchmarks for Linkerd and Envoy</b>
```

Comment:
A typo in the previous line. The full stop gets out of the paragraph. Rest looks good.

Reply:
Now placed full stop correctly